systems/subscriber/api-gateway/pkg/rest/router.go

```diff
@@ -4,21 +4,22 @@ import (
 	"fmt"
```

Comment:
There are some mismatches between the endpoints  defined in the postman specs and these implemented by this router. So maybe we can try to sync between these two first.